src/other/protocols/importAsset.js

```diff
@@ -103,7 +102,7 @@ const validateCsv = data =>
 export const validateAsset = (file) => {
   const assetType = getTypeFromMime(file.type) || getTypeFromExtension(file.name);
 
-  if (assetType !== 'network') { Promise.resolve(true); }
+  if (assetType !== 'network') { return Promise.resolve(true); }
```

Comment:
Same question with this...was it a bug?!

Reply:
Possibly a bug; I haven't checked master (yet). I couldn't add image assets before I added this.

Reply:
Nice spot! If not a bug now, one waiting to happen! Can we add a test?